### PR TITLE
259987-Update to .Net 4.8

### PR DESCRIPTION
**Pull request by ghost**
MINOR

Update .csproj and .config files for .net framework 4.8.  
Limit code support to C# 7.3.
(Directory.Build.props allows build process to automatically apply included values to each project in the solution, as opposed to having to do each separately)

### CODE REVIEW

File: paket.dependencies [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+content: none
+
+source http://artrepo.daptiv.com/artifactory/api/nuget/nuget-virtual
+
+nuget Common.Logging 3.0.0
+nuget Common.Logging.Core 3.0.0
+nuget FluentAssertions 3.2.1
+nuget log4net 2.0.3
+nuget Moq 4.2.1409.1722
+nuget NuGet.CommandLine
```
Comment (jaredreynolds) [inline paket.dependencies:11]: remove

**Review by jaredreynolds — CHANGES_REQUESTED** [review]
Remove paket.bootstrapper.exe.  Since paket.exe is using [magic mode](https://fsprojects.github.io/Paket/bootstrapper.html#Magic-mode), you should be good with just the single exe.  Also, make sure "paket.exe" is current (it's really bootstrapper).  In theory, the two exes you do have here should have been the same.

**Review by kirasglimmer — CHANGES_REQUESTED** [review]
For `.paket/paket.exe and .paket/paket.bootstrapper.exe`, use the "automagic" mode - aka, rename paket.bootstrapper.exe to paket.exe.  This is standard approach for smaller repositories that don't build very often.

Otherwise, looks good.

File: paket.dependencies (hunk 1 above)
Comment (ghost) [reply paket.dependencies:11]: These are needed for example and test projects

**Review by jaredreynolds — APPROVED** [review]

**Review by kirasglimmer — APPROVED** [review]

**Review by jtrinklein — APPROVED** [review]

**Review by jtrinklein — APPROVED** [review]

**Pull request merged**